includes/LST.php

```diff
@@ -923,4 +924,35 @@ protected static function callParserPreprocess( Parser $parser, $text, $page, $o
 			return $parser->preprocess( $text, $page, $options );
 		}
 	}
+
+	private static function softResetParser( Parser $parser ): void {
```

Comment:
Let's clarify that this is done for compatibility to not exceed limit counters while still allowing to reuse template DOM etc. caches